tests/commands/train_test.py

```diff
@@ -90,3 +92,87 @@ def test_train_args(self):
         with self.assertRaises(SystemExit) as cm:  # pylint: disable=invalid-name
             args = parser.parse_args(["train", "path/to/params"])
             assert cm.exception.code == 2  # argparse code for incorrect usage
+
+@DatasetReader.register('lazy-test')
+class LazyTestReader(DatasetReader):
```

Comment:
I _think_ having `Test` in this name is potentially confusing to pytest (e.g., it'd try to run methods on here as tests if they happened to also have `test` in their name).  Maybe call this `LazyFakeReader`?